style.css

```diff
@@ -8,29 +8,24 @@ body {
   margin: 0;
 }
 
-.centering-outer-container {
-  position: absolute;
-  display: table;
+.page-container {
   width: 100%;
-  height: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
-.centering-inner-container {
-  display: table-cell;
-  vertical-align: middle;
-  text-align: center;
-}
-
-.centering-content {
-  display: inline-block;
-}
-
-#content {
-  background-color: #fefefe;
-  max-width: 400px;
-  height: 400px;
-  padding: 20px;
+.content-box {
+  background-color: white;
```

Comment:
Thank you for teaching me this! I didn't have any idea. I'll make the change asap.